loader_hub/intercom/base.py

```diff
@@ -0,0 +1,93 @@
+"""Intercom reader."""
+from typing import List
+import requests
+import json
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from bs4 import BeautifulSoup
```

Comment:
you may have to do lazy importing of bs4 for tests to pass. see `loader_hub/web/beautiful_soup_web/base.py` as an example 

(i know it's pretty hacky)